core/rounds.py

```diff
@@ -0,0 +1,20 @@
+from connexion import NoContent
+from core.data_operations import round_data
+
+# from flask import request
+
+
+def get_rounds_for_fund(fund_id: str):
+    rounds = round_data.ROUNDS_DUMMY_DAO.get_all(fund_id)
```

Comment:
Re comment on the DAO, maybe this should be more accurately be a called filter as a method as used here perhaps.

Reply:
👌